deploy/GOV/003-PostDeploymentChecks.ts

```diff
@@ -74,20 +74,14 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const lockingAddress = await factory.locking();
   const locking = await ethers.getContractAt("Locking", lockingAddress);
 
-  const mentoLabsTreasuryAddress = await factory.mentoLabsTreasuryTimelock();
-  const mentoLabsTreasury = await ethers.getContractAt("TimelockController", mentoLabsTreasuryAddress);
 
   // mentoToken checks
   const mentoLabsMultisigBalance = await mentoToken.balanceOf(MENTO_LABS_MULTISIG);
   ok(
-    mentoLabsMultisigBalance.toString() === ethers.parseEther("80000000").toString(),
+    mentoLabsMultisigBalance.toString() === ethers.parseEther("200000000").toString(),
```

Comment:
Why did we change the expected balance here?

Reply:
since we don't have a seperate timelock contract to hold timelocked assets, the allocation for that goes directly multisig

Reply:
Got it, thanks 👍